lib/client.js

```diff
@@ -155,27 +156,31 @@ const PodiumClient = class PodiumClient extends EventEmitter {
     }
 
     js() {
-        return Array.from(this[_resources])
+        return [].concat.apply([], Array.from(this[_resources])
```

Comment:
I know this is because of flattening, but man this looked really weird to me at first.

Reply:
Its fast though ;)

On the serious side; all this looping will be removed in version 5.x of the client.